wp-content/themes/mstoday/functions.php

```diff
@@ -352,3 +352,37 @@ function mstoday_subscribe_button( $location = null ) {
 
     }
 } 
+
+/**
+ * Add relevant data attribute to scripts that should be ignored by 
+ * CloudFlare's Rocket Loader functionality
+ * 
+ * @link Rocket Loader docs: https://support.cloudflare.com/hc/en-us/articles/200169436-How-can-I-have-Rocket-Loader-ignore-specific-JavaScripts-
+ * @link GitHub issue: https://github.com/INN/umbrella-mstoday/issues/81
+ * 
+ * @param str $tag The <script> tag for the enqueued script.
+ * @param str $handle The script's registered handle.
+ * @param str $src The script's source URL.
+ * 
+ * @return str $tag The <script> tag for the enqueued script.
+ */
+function mstoday_rocket_load_ignored_scripts( $tag, $handle, $src ) {
+
+	// array full of script handles that CF RL should ignore
+	$scripts_to_ignore = array(
+		'load-more-posts'
```

Comment:
I think we need to add jquery here.

The docs say:
> If the script you want Rocket Loader to ignore has dependency on other JavaScript(s) on the page, those depedencies must also have the data-cfasync="false" attribute added.

`jquery` is the only listed dependency of this script:

https://github.com/INN/largo/blob/512da701664b329f2f92244bbe54880a6e146431/inc/ajax-functions.php#L25-L31

Reply:
Ah, good catch and good idea!

Reply:
@benlk Added in https://github.com/INN/umbrella-mstoday/pull/82/commits/49aabb97a3b93c1ab8ab0651e24f9b167ddc816e